spec/20-http_request_header_format.md

```diff
@@ -100,13 +100,17 @@ trace-flags      = 2HEXDIGLC   ; 8 bit flags. Currently, only one bit is used. S
 
 #### trace-id
 
-This is the ID of the whole trace forest and is used to uniquely identify a <a href="#dfn-distributed-traces">distributed trace</a> through a system. It is represented as a 16-byte array, for example, `4bf92f3577b34da6a3ce929d0e0e4736`. All bytes as zero (`00000000000000000000000000000000`) is considered an invalid value.
+This is the ID of the whole trace forest and is used to uniquely identify a <a href="#dfn-distributed-traces">distributed trace</a> through a system.
+It is represented as a 16-byte array, for example, `4bf92f3577b34da6a3ce929d0e0e4736`.
+All bytes as zero (`00000000000000000000000000000000`) is considered an invalid value.
 
-If the `trace-id` value is invalid (for example if it contains non-allowed characters or all zeros), vendors MUST ignore the `traceparent`.
+The value of `trace-id` SHOULD be globally unique.
+It is RECOMMENDED that implementers randomly (or pseudo-randomly) generate the `trace-id` field, and set the corresponding [random trace id flag](#random-trace-id-flag).
```

Comment:
it is not a concern for this PR. The conformance section that is automatically generated doesn't mention the "RECOMMENDED" keyword, even though it exists in the spec the section refers to.

![image](https://user-images.githubusercontent.com/9950081/163039956-4eef7ab5-ca76-4588-908d-c70ec143f69b.png)

I wonder if this can be fixed, @plehegar do you know?

If this cannot be fixed we can switch to SHOULD as they are interchangeable with RECOMMENDED as per my understanding